common/v2/features/Settings/Settings.tsx

```diff
@@ -44,10 +44,15 @@ const SettingsTabs = styled(TabsNav)`
 `;
 
 function renderAccountPanel() {
+  const { accounts } = useContext(StoreContext);
   return (
     <FlippablePanel>
       {({ flipped }) =>
-        flipped ? <p>Add Account</p> : <AccountList deletable={true} copyable={true} />
+        flipped ? (
+          <p>Add Account</p>
```

Comment:
We probably have a translation string for this somewhere :dancer:

Reply:
I would prefer to change this in a separate translations PR.
If I start I would want to translate the entire view, so I prefer to keep it out of scope atm.